sympy/logic/boolalg.py

```diff
@@ -3118,3 +3122,51 @@ def simplify_patterns_xor():
                       And(Lt(a, Max(b, c)), Ge(a, Min(b, c)))),
                      )
     return _matchers_xor
+
+
+# Classes corresponding to logic gates
+# Used in gateinputcount method
+BooleanGates = (And, Or, Xor, Nand, Nor, Not, Xnor, ITE)
+
+def gateinputcount(expr):
```

Comment:
Will this function automatically show up in the docs somewhere or does a reference to it need be added in an rst file?

Reply:
Good question! Will try to figure out.